Formula/fastfec.rb

```diff
@@ -16,29 +17,40 @@ class Fastfec < Formula
 
   depends_on "pkg-config" => :build
   depends_on "zig" => :build
-  depends_on "pcre"
-  uses_from_macos "curl"
+
+  on_macos do
+    # Zig attempts to link with `libpcre.a` on Linux.
+    # This fails because it was not compiled with `-fPIC`.
+    # Use Homebrew PCRE on Linux when upstream resolves
+    #   https://github.com/ziglang/zig/issues/14111
+    # Don't forget to update the `install` method.
+    depends_on "pcre"
```

Comment:
Can it use pcre2?

Reply:
Unfortunately not.

Reply:
If there isn't an issue yet, could you make one for that too?

Reply:
https://github.com/washingtonpost/FastFEC/issues/57